test_unique_character_counter.py

```diff
@@ -0,0 +1,41 @@
+from collections import Counter
+
+
+class UniqueCharacterCounter:
```

Comment:
This file is called "test....". But this is not testing. This is the main functionality. Move to a different file.

Test files should exist only for testing